debug/src/devtools/10/vnode.js

```diff
@@ -0,0 +1,95 @@
+import { Fragment } from 'preact';
+
+/**
+ * Check if a `vnode` is the root of a tree
+ * @param {import('../../internal').VNode} vnode
+ * @returns {boolean}
+ */
+export function isRoot(vnode) {
+	return vnode._parent == null && vnode.type === Fragment;
```

Comment:
Makes me wonder if we need a special indicator for this node

Reply:
Yeah, true, we may need something to mark root nodes as roots instead of plain `Fragments` in the future.